.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
+
+    steps:
```

Comment:
Could you please add reuse compliance step here?

https://github.com/marketplace/actions/reuse-compliance-check

Reply:
Added in a `reuse.yml` file — I think it's probably best to separate them, so that PRs will report separately which of the two failed. We could also include separate badges on the readme, if we'd like.

Reply:
Hah, and it's failing for a very relevant reason... I'll fix that too.

![Screen Shot 2020-11-19 at 9 49 49 AM](https://user-images.githubusercontent.com/1848368/99703949-958f2700-2a4c-11eb-8e6d-c7cbf604b6ab.png)